docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -59,19 +59,30 @@ image::images/view-alert-details.png[Shows the Event Renderer button, 200]
 
 The Alert details flyout appears and offers several options for viewing alert details:
 
-* *Summary*: Shows an aggregated view of alert details. Alerts that have been enriched with `threat.indicator` data also display the *threat summary* section, which is an additional section located beneath the alert summary. In the *threat summary* section, you can view mapped data for the following `threat.indicator` subfields:
-** `matched.field`
-** `matched.type`
-** `source (threat.indicator.provider)`
-** `first_seen`
-** `last_seen`
+* *Summary*: Shows an aggregated view of alert details, plus two types of threat intelligence data: indicator rule enrichments and investigation time enrichments.
```

Comment:
I remember @shimonmodi mentioning that we shouldn't use these phrases (indicator match enrichments and investigation time enrichments), but I am not entirely sure exactly what we agreed on instead - we might want to check with @shimonmodi and @devonakerr.

Reply:
For context, here is the latest copy for those pages: https://github.com/elastic/kibana/pull/105701